weblab/templates/entities/entity_version.html

```diff
@@ -40,12 +41,17 @@ <h2 id="entityversionname">
             <a href="{{ entity|url_tag_version:version }}">Add tag</a>.
             <form method="POST" action="">
               Change visibility: {{ form.visibility }}
-              <img src="{% static 'img/info.png' %}" alt="help" title="{{ entity.VISIBILITY_HELP }}" />.
+              <img src="{% static 'img/info.png' %}" alt="help" title="{{ entity.VISIBILITY_HELP }}" />
               <span id="versionVisibilityAction"></span>
-            </form>
-          {% else %}
+              <!-- SK: put in permission -->
+              Run experiments:
+                <a title="Run experiments" href="{{ entity|url_runexperiments }}">
+                    <img src="{% static 'img/batch.png' %}" alt="Run experiments:" title="Run experiments using this model"/>
```

Comment:
`s/model/{{ entity.entity_type }}/`

Reply:
Sorry not sure what you mean ??

Reply:
You have the `title` as "Run experiments using this model" - instead of model you should insert either model or protocol depending on the entity_type.

Reply:
Doh !

Sorted